src/ports/rt-kernel/pnal_snmp.c

```diff
@@ -0,0 +1,96 @@
+/*********************************************************************
+ *        _       _         _
+ *  _ __ | |_  _ | |  __ _ | |__   ___
+ * | '__|| __|(_)| | / _` || '_ \ / __|
+ * | |   | |_  _ | || (_| || |_) |\__ \
+ * |_|    \__|(_)|_| \__,_||_.__/ |___/
+ *
+ * www.rt-labs.com
+ * Copyright 2020 rt-labs AB, Sweden.
+ *
+ * This software is dual-licensed under GPLv3 and a commercial
+ * license. See the file LICENSE.md distributed with this software for
+ * full license information.
+ ********************************************************************/
+
+#include "pf_includes.h"
+#include "pnal_snmp.h"
+#include "lldp-mib.h"
+#include <lwip/apps/snmp.h>
+#include <lwip/apps/snmp_mib2.h>
+#include <lwip/apps/snmp_threadsync.h>
+
+os_snmp_t os_snmp;
+
+/**
+ * Called right after writing a variable
+ *
+ * @param oid           Object ID array, e.g
+ * @param oid_len
+ * @param callback_arg
+ */
+static void os_snmp_write_callback (
```

Comment:
Should it be `pnal_snmp_xxx `instead of `os_snmp_xxx` ?
(Most of functions in pnal has not yet been renamed from os_xxx)

Reply:
Done.